src/requests/getImages.js

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+
+const getImages = (query) => {
+    if (!query) {
+        return Promise.resolve([]);
+    } else {
+        return axios
+            .get(`https://images-api.nasa.gov/search?q=${query}`)
+            .then((response) => {
+            const imageResults = response.data.collection.items
+            const parsedImages = imageResults.filter(imageResult => imageResult.data[0].media_type === "image")
+            const images = parsedImages.map(image => image.links[0].href)
```

Comment:
Challenge - try using a reduce instead of a map and filter, can you refactor it?